doc/voq/lag_hld.md

```diff
@@ -545,15 +546,50 @@ Shown below is the new attribute of SAI_OBJECT_TYPE_LAG object that is used for
 
 ## 8 Example Show Commands
 
+The system lag information are taken from **SYSTEM_LAG_TABLE** of **CHASSIS_APP_DB** from supervisor card.
+
+The show commands for system lags are under **show chassis** commands group as show below
+
+```
+admin@Linecard2:~$ show chassis -h
+Usage: show chassis [OPTIONS] COMMAND [ARGS]...
+
+  Chassis commands group
+
+Options:
+  -h, -?, --help  Show this message and exit.
+
+Commands:
+  modules           Show chassis-modules information
+  system-lags       Show VOQ system lags information
+  system-neighbors  Show VOQ system neighbors information
+  system-ports      Show VOQ system ports information
+admin@Linecard2:~$ 
 ```
-admin@sonic:~$ show interfaces portchannel system
-Flags: A - active, I - inactive, Up - up, Dw - Down, N/A - not available,
-       S - selected, D - deselected, * - not synced, L - local switch
-  No.  Switch ID    LAG Name      Protocol   Ports
------  ----------   -----------   ---------- --------------
-    1  0(L}         PortChannel1  SystemLAG  Slot1|Asic0|Ethernet1(S)  Slot1|Asic0|Ethernet2(S)
-    2  6            PortChannel1  SystemLAG  Slot2|Asic0|Ethernet1(S)  Slot2|Asic0|Ethernet2(S)
+### Syntax
 ```
+admin@Linecard2:~$ show chassis system-lags -h
+Usage: show chassis system-lags [OPTIONS] [SYSTEMLAGNAME]
+
+  Show VOQ system lags information
+
+Options:
+  -d, --display [all]  Show internal interfaces  [default: all]
+  -n, --namespace []   Namespace name or all
```

Comment:
Can you elabarote what will the difference in output between the commands `show chassis system-lags` and `show chassis system-lags -n asic0` ?

Reply:
The intention was to use this to display the lags from a specific asic. The system lag name includes the DEVICE_METADATA['localhost']['asic_name'] which is expected to be same as the namespace name. If not, this will not work. 
Do you think we can remove this option?

Reply:
Fixed. Since the system lag info is retrieved form chassis app db, which in global name space in supervisor card (i.e., it is namespace agnostic), instead of using namespace option, an option is introduced to take the asic_name (this need not be namespace name) which is used for filtering the output for the given specific asic.